client/src/view/info-pages.tsx

```diff
@@ -54,25 +30,20 @@ export const INFO_PAGES: JSX.Element[] = [
     (
         <div>
             <p>
-                As the story goes, <strong>galapa</strong> means something like
-                "family" in the local dialect and a <strong>hexalot</strong>,
-                is a unique beehive pattern of surface spots where a *gotchi* lives.
+                Every "hexalot" is a place with a completely unique pattern of spots
```

Comment:
"a place"? Be more specific

Reply:
A "hexalot" is a region of the island shaped like a honeycomb,
with 127 spots which are either land or water.

There are no two patterns alike,
and since hexalots overlap each one is forever connected to its neighbors.